packages/teleport-shared/src/node-handlers/node-to-html/index.ts

```diff
@@ -63,7 +63,10 @@ const generateElementNode = (
 ) => {
   const { dependencies, templateLookup } = params
   const { elementType, name, key, children, attrs, dependency, events } = node.content
-  const htmlNode = createHTMLNode(elementType)
+  const htmlNode =
+    dependency && dependency.type === 'local'
```

Comment:
I think it's enough check for dependency, dependency type package should also count as custom element